lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -1473,7 +1473,38 @@ padding:1px;
 .edit-col span {
 color: #666;
 }
+.calendar-div{
+max-width: 550px;
+border: solid #999 1px;
+border-radius: 0px 0px 10px 10px;
+}
+
+.eventAttachmentList{
+display: inline-table;
+}
 
+eventAttachmentItem{
+list-style-type: none;
+}
+.fc-event{
+cursor: pointer;
+margin-bottom: 6px; /*to add space between two events*/
+}
+.fc-today-button{
+background-color: #eee !important;
```

Comment:
Why are we using all this !important styling? It's not great css practice - can't we do it another way?

There are more uses of it below.